config/sync/core.entity_view_display.node.stanford_opportunity.default.yml

```diff
@@ -568,10 +568,14 @@ third_party_settings:
                 entity: layout_builder.entity
                 view_mode: view_mode
               formatter:
-                type: entity_reference_label
+                type: cshs_group_by_root
```

Comment:
Units were should not use this formatter display either. Units should not be grouped. They should be just a list.

Reply:
Ok. Fixing it

Reply:
@pookmish I fixed the Units. I updated the css as well.